### YamlMime:ManagedReference
items:
- uid: com.microsoft.azure.management.powerbi._error_exception
  id: _error_exception
  parent: com.microsoft.azure.management.powerbi
  children:
  - com.microsoft.azure.management.powerbi._error_exception.body()
  - com.microsoft.azure.management.powerbi._error_exception.ErrorException(final String,final Response<ResponseBody>)
  - com.microsoft.azure.management.powerbi._error_exception.ErrorException(final String,final Response<ResponseBody>,final Error)
  href: com.microsoft.azure.management.powerbi._error_exception.yml
  langs:
  - java
  name: ErrorException
  nameWithType: ErrorException
  fullName: com.microsoft.azure.management.powerbi.ErrorException
  type: Class
  source:
    remote: &o0
      path: azure-mgmt-powerbi/src/main/java/com/microsoft/azure/management/powerbi/ErrorException.java
      branch: master
      repo: https://github.com/Azure/azure-sdk-for-java
    path: azure-mgmt-powerbi/src/main/java/com/microsoft/azure/management/powerbi/ErrorException.java
    startLine: 17
  package: com.microsoft.azure.management.powerbi
  summary: <p>Exception thrown for an invalid response with <xref href="com.microsoft.azure.management.powerbi._error" data-throw-if-not-resolved="false"></xref> information. </p>
  syntax: &o1
    content: public class ErrorException
  inheritance:
  - java.lang.Object
  - RestException
- uid: com.microsoft.azure.management.powerbi._error_exception.body()
  id: body()
  parent: com.microsoft.azure.management.powerbi._error_exception
  href: com.microsoft.azure.management.powerbi._error_exception.yml
  langs:
  - java
  name: body()
  nameWithType: ErrorException.body()
  fullName: Error com.microsoft.azure.management.powerbi.ErrorException.body()
  overload: com.microsoft.azure.management.powerbi._error_exception.body*
  type: Method
  source:
    remote: *o0
    path: azure-mgmt-powerbi/src/main/java/com/microsoft/azure/management/powerbi/ErrorException.java
    startLine: 40
  package: com.microsoft.azure.management.powerbi
  syntax:
    content: public Error body()
    return:
      type: com.microsoft.azure.management.powerbi._error
- uid: com.microsoft.azure.management.powerbi._error_exception.ErrorException(final String,final Response<ResponseBody>)
  id: ErrorException(final String,final Response<ResponseBody>)
  parent: com.microsoft.azure.management.powerbi._error_exception
  href: com.microsoft.azure.management.powerbi._error_exception.yml
  langs:
  - java
  name: ErrorException(final String message, final Response<ResponseBody> response)
  nameWithType: ErrorException.ErrorException(final String message, final Response<ResponseBody> response)
  fullName: com.microsoft.azure.management.powerbi.ErrorException.ErrorException(final String message, final Response<ResponseBody> response)
  overload: com.microsoft.azure.management.powerbi._error_exception.ErrorException*
  type: Constructor
  source:
    remote: *o0
    path: azure-mgmt-powerbi/src/main/java/com/microsoft/azure/management/powerbi/ErrorException.java
    startLine: 24
  package: com.microsoft.azure.management.powerbi
  summary: >-
    <p>Initializes a new instance of the <xref href="com.microsoft.azure.management.powerbi._error_exception" data-throw-if-not-resolved="false"></xref> class.</p>

    <p></p>
  syntax:
    content: public ErrorException(final String message, final Response<ResponseBody> response)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message or the response content if a message is not available </p>
    - id: response
      type: 4f402e61
      description: <p>the HTTP response </p>
- uid: com.microsoft.azure.management.powerbi._error_exception.ErrorException(final String,final Response<ResponseBody>,final Error)
  id: ErrorException(final String,final Response<ResponseBody>,final Error)
  parent: com.microsoft.azure.management.powerbi._error_exception
  href: com.microsoft.azure.management.powerbi._error_exception.yml
  langs:
  - java
  name: ErrorException(final String message, final Response<ResponseBody> response, final Error body)
  nameWithType: ErrorException.ErrorException(final String message, final Response<ResponseBody> response, final Error body)
  fullName: com.microsoft.azure.management.powerbi.ErrorException.ErrorException(final String message, final Response<ResponseBody> response, final Error body)
  overload: com.microsoft.azure.management.powerbi._error_exception.ErrorException*
  type: Constructor
  source:
    remote: *o0
    path: azure-mgmt-powerbi/src/main/java/com/microsoft/azure/management/powerbi/ErrorException.java
    startLine: 35
  package: com.microsoft.azure.management.powerbi
  summary: >-
    <p>Initializes a new instance of the <xref href="com.microsoft.azure.management.powerbi._error_exception" data-throw-if-not-resolved="false"></xref> class.</p>

    <p></p>
  syntax:
    content: public ErrorException(final String message, final Response<ResponseBody> response, final Error body)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message or the response content if a message is not available </p>
    - id: response
      type: 4f402e61
      description: <p>the HTTP response </p>
    - id: body
      type: 93de0ef9com.microsoft.azure.management.powerbi._error
      description: <p>the deserialized response body </p>
references:
- uid: 3d93591e
  spec.java:
  - name: final String
    fullName: final String
- uid: 4f402e61
  spec.java:
  - name: final Response<ResponseBody>
    fullName: final Response<ResponseBody>
- uid: com.microsoft.azure.management.powerbi._error_exception.ErrorException*
  name: ErrorException
  nameWithType: ErrorException.ErrorException
  fullName: com.microsoft.azure.management.powerbi.ErrorException.ErrorException
  package: com.microsoft.azure.management.powerbi
- uid: 93de0ef9com.microsoft.azure.management.powerbi._error
  spec.java:
  - name: 'final '
    fullName: 'final '
  - uid: com.microsoft.azure.management.powerbi._error
    name: Error
    fullName: com.microsoft.azure.management.powerbi.Error
    href: com.microsoft.azure.management.powerbi._error.yml
- uid: com.microsoft.azure.management.powerbi._error_exception.body*
  name: body
  nameWithType: ErrorException.body
  fullName: Error com.microsoft.azure.management.powerbi.ErrorException.body
  package: com.microsoft.azure.management.powerbi
- uid: com.microsoft.azure.management.powerbi._error_exception
  parent: com.microsoft.azure.management.powerbi
  href: com.microsoft.azure.management.powerbi._error_exception.yml
  name: ErrorException
  nameWithType: ErrorException
  fullName: com.microsoft.azure.management.powerbi.ErrorException
  type: Class
  summary: <p>Exception thrown for an invalid response with <xref href="com.microsoft.azure.management.powerbi._error" data-throw-if-not-resolved="false"></xref> information. </p>
  syntax: *o1
- uid: com.microsoft.azure.management.powerbi._error
  parent: com.microsoft.azure.management.powerbi
  href: com.microsoft.azure.management.powerbi._error.yml
  name: Error
  nameWithType: Error
  fullName: com.microsoft.azure.management.powerbi.Error
  type: Class
  summary: <p>The <xref href="com.microsoft.azure.management.powerbi._error" data-throw-if-not-resolved="false"></xref> model. </p>
  syntax:
    content: public class Error
